### ISSUE TITLE
Wrong host labels fields used with v1.29.1

### ISSUE THREAD

**Issue by mihkelvain** [netdata/netdata#10646]
##### Bug report summary
After upgrading to v1.29.1 from v1.26.0 os_* related labels are set to "unknown":

v.126.0 example:
```
curl http://<central-netdata>:19999/host/<agent-hostname1>/api/v1/info

{
  "version": "v1.26.0",
  "uid": "74920492-57b5-11ea-88c0-1402ec772fcc",

...

},
  "os_name": "CentOS Linux",
  "os_id": "centos",
  "os_id_like": "rhel fedora",
  "os_version": "7 (Core)",
  "os_version_id": "7",
  "os_detection": "/etc/os-release",
  "cores_total": "40",
  "total_disk_space": "2400328966144",
...
```

With v1.29.1 on agent
```
curl http://<central-netdata>:19999/host/<agent-hostname1>/api/v1/info

{
  "version": "v1.29.1",
  "uid": "b94bfe56-7057-11eb-b67d-005056bd08d8",

...

  },
  "os_name": "unknown",
  "os_id": "unknown",
  "os_id_like": "unknown",
  "os_version": "unknown",
  "os_version_id": "unknown",
  "os_detection": "unknown",
  "cores_total": "16",
  "total_disk_space": "343597383680",
....
```

Instead distro related information is written to container_os* related fields.:
```
...
  "container_os_name": "CentOS Linux",
  "container_os_id": "centos",
  "container_os_id_like": "rhel fedora",
  "container_os_version": "7 (Core)",
  "container_os_version_id": "7",
...
```

##### OS / Environment
CentOS 7 and 8
No docker. Static binary builds from github

##### Netdata version
v1.29.1

##### Expected behavior
Version v1.29.1 should also advertise distro/operating system related information to os_* fields like with older versions. To me makes no sense to use container_os* fields.

**Comment by ilyam8** [comment]
Netdata Agent installed directly on the host.

- Current master

```cmd
[ilyam@pc ~]$ curl -s http://127.0.0.1:19999/api/v1/info | grep "os_*"
	"mirrored_hosts": [
	"mirrored_hosts_status": [
	"os_name": "unknown",
	"os_id": "unknown",
	"os_id_like": "unknown",
	"os_version": "unknown",
	"os_version_id": "unknown",
	"os_detection": "unknown",
	"container_os_name": "Manjaro Linux",
	"container_os_id": "manjaro",
	"container_os_id_like": "arch",
	"container_os_version": "unknown",
	"container_os_version_id": "unknown",
	"host_labels": {
		"_os_name": "unknown",
		"_os_version": "unknown",
```

- After revert https://github.com/netdata/netdata/pull/10569

```cmd
[ilyam@pc ~]$ curl -s http://127.0.0.1:19999/api/v1/info | grep "os_*"
	"mirrored_hosts": [
	"mirrored_hosts_status": [
	"os_name": "Manjaro Linux",
	"os_id": "manjaro",
	"os_id_like": "arch",
	"os_version": "unknown",
	"os_version_id": "unknown",
	"os_detection": "/etc/os-release",
	"container_os_name": "none",
	"container_os_id": "none",
	"container_os_id_like": "none",
	"container_os_version": "none",
	"container_os_version_id": "none",
	"host_labels": {
		"_os_name": "Manjaro Linux",
		"_os_version": "unknown",
```

**Comment by mihkelvain** [comment]
For me labels are ok after this: https://github.com/mihkelvain/netdata/commit/7e40bce55e96cc02686f3741d218daaf858deaa5

Idk why /host/etc/os-release is used instead of /etc/os-release

**Comment by ilyam8** [comment]
@mihkelvain we use `/host/` to determine **host** system parameters.

See [Create a new netdata agent container](https://learn.netdata.cloud/docs/agent/packaging/docker#create-a-new-netdata-agent-container), we mount hosts `/etc/os-release` to containers `/host/etc/os-release`

**Comment by ilyam8** [comment]
Thanks for reporting @mihkelvain. The bug is fixed in #10647.
